doc/Alerting/Templates.md

```diff
@@ -116,7 +116,6 @@ Unique-ID: {{ $alert->uid }}
 Rule: @if ($alert->name) {{ $alert->name }} @else {{ $alert->rule }} @endif
 @if ($alert->faults) Faults:
 @foreach ($alert->faults as $key => $value)
-  #{{ $key }}: {{ $value['string'] }}
```

Comment:
Why did you remove this?

Reply:
The lines starting with hash tags cause the parser to throw a syntax error.  If they are supposed to be comments and you would like to keep the line then I can amend it to use the {{-- comment --}} which I think is the laravel comment formatting.

Reply:
I think the hash is supposed to be a literal hash in the result.  No special meaning.
Really odd that it would cause a syntax error.